modules/syslogformat/syslog-format.c

```diff
@@ -572,11 +572,6 @@ log_msg_parse_rfc5424_date_unnormalized(LogMessage *self, const guchar **data, g
   *data = src;
   *length = left;
   return TRUE;
-error:
-  /* no recognizable timestamp, use current time */
-
-  self->timestamps[LM_TS_STAMP] = self->timestamps[LM_TS_RECVD];
```

Comment:
If you look at the caller of this function, it does the same assignment in case this function returns FALSE, so we basically assigned the same value twice.